README.md

```diff
@@ -21,6 +21,13 @@
 - You can help us out by resolving TODOs and replying on issues
 - Of course, all feedback, positive and negative, also really helps to improve the project
 
+### How to run the project
+* Clone the repository
+* Navigate into it and run `vapor update` ([make sure to install Vapor](https://docs.vapor.codes/3.0/install/macos/))
+* To add the test database: click on the "Run" target in the top left hand side and add the environment variable `DB_URL` with a value of `mongodb://swifttube-test:test123@ds113445.mlab.com:13445/swifttube-test`
```

Comment:
Swifttube schema already has the DB Url. We can also remove the Run schema not to confuse first-timers.

Reply:
Ah didn't realise, I've been manually inputting it to the run scheme every time 😂